README.md

```diff
@@ -1,28 +1,59 @@
 <a href="https://github.com/hypothesis/report/actions/workflows/ci.yml?query=branch%3Amain"><img src="https://img.shields.io/github/workflow/status/hypothesis/report/CI/main"></a>
 <a><img src="https://img.shields.io/badge/python-3.8-success"></a>
 <a href="https://github.com/hypothesis/report/blob/main/LICENSE"><img src="https://img.shields.io/badge/license-BSD--2--Clause-success"></a>
-<a href="https://github.com/hypothesis/cookiecutters/tree/main/pyramid-app"><img src="https://img.shields.io/badge/cookiecutter-pyramid--app-success"></a>
+<a href="https://github.com/hypothesis/cookiecutters/tree/main/pyapp"><img src="https://img.shields.io/badge/cookiecutter-pyapp-success"></a>
 <a href="https://black.readthedocs.io/en/stable/"><img src="https://img.shields.io/badge/code%20style-black-000000"></a>
 
 # Report
 
-The internal global reporting product for Hypothesis
+The internal global reporting product for Hypothesis.
+
+Used for internal purposes only.
+
+## Environment variables
+
+
+| Name                              | Example                                | Notes                                           | 
+|-----------------------------------|----------------------------------------|-------------------------------------------------|
+| `DATABASE_URL`                    | `postgresql://user:pw@host/report`        | Postgres DSN for the report DB                                   |
+
+
+On top of the service own environment variables these are the metabase variables that we use:
+
+
+| Name                              | Example                                | Notes                                           | 
+|-----------------------------------|----------------------------------------|-------------------------------------------------|
+| `MB_DB_DBNAME`                    | `metabase`        | Metabase database name                                   |
+| `MB_DB_HOST`                    | `localhost`        | Metabase database host                                   |
+| `MB_DB_PASS`                    | `pass`        | Metabase database password                                   |
+| `MB_DB_PORT`                    | `5432`        | Metabase database port                                   |
+| `MB_DB_TYPE`                    | `postgres`        | Metabase database type. We use `postgres`. |
+| `MB_DB_USER`                    | `user`        | Metabase database user                                   |
+
+In addition, we are also providing some custom Java options
+
+| Name           | Value                                                | Description           |
+|----------------|------------------------------------------------------|-----------------------|
+| `JAVA_OPTS`    |-Dlog4j.configurationFile=file://conf/report-log4j2.xml| Custom log4j config   |
+
+The full list of supported variables by metabase can be found here:
+
+https://www.metabase.com/docs/latest/configuring-metabase/environment-variables.html
```

Comment:
This is the contents of `.cookiecutter/includes/README/head.md` that is [already on main](https://github.com/hypothesis/report/blob/main/.cookiecutter/includes/README/head.md) but that wasn't actually getting into the real `README.md` on `main` because the `pyramid-app` cookiecutter's `README.md` didn't support the `head.md` include. That was fixed by https://github.com/hypothesis/cookiecutters/pull/87, hence this text is now getting included in `README.md`